src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -77,6 +78,9 @@ export default App.extend(extend({
   onCommentAdded(model) {
     this.activityCollection.add(model);
   },
+  onAttachmentAdded(model) {
```

Comment:
This function name is really similar to the existing `onAddAttachment()` one. Maybe we should rename these to something more specific like `onWsCommentAdded` and `onWsAttachmentAdded()` or something similar?